keyboards/akb/eb46/info.json

```diff
@@ -0,0 +1,55 @@
+{
+    "keyboard_name": "eb46",
+    "maintainer": "e11i0t23",
+    "width": 12,
+    "hight": 4,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "x": 0, "y": 0 },
```

Comment:
Your layout macro

https://github.com/qmk/qmk_firmware/blob/a1f8cc1699ec4eb880176ef2c871bcbc269a8e27/keyboards/akb/eb46/eb46.h#L29-L34

defines 46 keys, but you only have 43 here.


This needs to be corrected for QMK Configurator's visual rendering to work.